client/src/components/App.jsx

```diff
@@ -7,8 +7,6 @@ import Ratings from './RatingsAndReviews/RatingsAndReviewsContainer';
 import RelatedItems from './RelatedItemsAndComparisons/RelatedItemsAndComContainer';
 
 export default function App() {
-  const [allProducts, setAllProducts] = useState();
-  const [currentProduct, setCurrentProduct] = useState();
   const [initProd, setInitProd] = useState({});
   const fetchProductData = () => {
     Promise.all([
```

Comment:
We should still be able to use initProd but later on please refactor this later so that styles response is stored in a separate state